### PR TITLE
Include calico.py, ipam.py in coverage reports

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

**Comment by TrimBiggs** [comment]
LGTM

File: tests/unit/test_cni_plugin.py [hunk 1]
```diff
@@ -355,6 +355,34 @@ def test_assign_ip_no_ip(self):
         e = err.exception
         assert_equal(e.code, ERR_CODE_GENERIC)
 
+    def test_assign_ip_invalid_ip4(self):
```
Comment (TrimBiggs) [inline tests/unit/test_cni_plugin.py]: Would be nice to have docstrings for these tests, but looks like most tests here don't have them either :\

**Comment by TrimBiggs** [comment]
LGTM with one minor nitpicky comment

File: tests/unit/test_cni_plugin.py (hunk 1 above)
Comment (caseydavenport) [inline tests/unit/test_cni_plugin.py]: Yeah you're right.

**Pull request merged**